### PR TITLE
612 author page v2

### PR DESCRIPTION
**Pull request by norisalsaadie**
closes https://github.com/libero/reviewer/issues/612

### CODE REVIEW

File: src/core/locales/en/author-guide.json [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+        "p1": "A fee of $2,500* is collected for published research papers; however, authors with insufficient funding to pay the fee are eligible for a fee waiver.",
+        "p2": "* If the fee is invoiced to an individual, rather than a business, then VAT of 20% is added to the fee.",
+        "p3": "The publication fee (also referred to as an \"open-access\" or \"article-processing\" fee) is a charge collected by many open-access publishers to help recover the costs of publication while providing free and open access for readers to use and re-use results. eLife provides open access to all content.",
+        "p4": "Questions related to our introduction of publication fees are addressed below.",
+        "p5": "For background and more information, please also read:",
+        "bullet1": "<a href=\"https://elifesciences.org/elife-news/inside-elife-setting-fee-publication\" target=\"_blank\" class=\"sc-gojNiO kiqxuj\">Inside eLife: Setting a fee for publication</a>",
+        "bullet2": "<a href=\"https://elifesciences.org/content/5/e21230\" target=\"_blank\" class=\"sc-gojNiO kiqxuj\">Editorial: Building a sustainable future for eLife</a>",
```
Comment (will-byrne) [inline src/core/locales/en/author-guide.json:467]: Classnames copied from xpub need to be changed to reviewer classses.

File: src/static-pages/styles/required-info-table.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        border-style: solid;
+        border-color: rgb(224, 224, 224);
+        border-image: initial;
+      }
+    }
+  
+    ul {
+      list-style: none;
+      padding: 0px;
+      // margin: -12px 0px 0px;
```
Comment (will-byrne) [inline src/static-pages/styles/required-info-table.scss:34]: comment accidentally commited

File: src/static-pages/components/ArticleTypes.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Interwave from 'interweave';
```
Comment (will-byrne) [inline src/static-pages/components/ArticleTypes.tsx:2]: typo on Interweave, this was quite confusing

File: src/static-pages/components/ArticleTypes.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+import { useTranslation } from 'react-i18next';
+import { Paragraph } from '../../ui/atoms';
+
+const ArticleTypes = (): JSX.Element => {
+    const { t } = useTranslation('author-guide');
+
+    return (
+        <div className="author-guide-content">
```
Comment (will-byrne) [inline src/static-pages/components/ArticleTypes.tsx:10]: className should follow the BEM convention we use in the other front end components
`author-guide__content`

File: src/static-pages/components/EditorialProcess.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            <Paragraph type="writing">
+                <Interwave content={t('editorial.full-sub.p1')} />
+            </Paragraph>
+
+            <h2>{t('editorial.revised-sub.heading')}</h2>
+            <Paragraph type="writing">
+                <Interwave content={t('editorial.revised-sub.p1')} />
+            </Paragraph>
+
+            <h2 id="scoop_protection">{t('editorial.faqs.heading')}</h2>
```
Comment (will-byrne) [inline src/static-pages/components/EditorialProcess.tsx:65]: is the `id` here needed?

File: src/static-pages/components/InitialSubmission.tsx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+import Interwave from 'interweave';
+import { useTranslation } from 'react-i18next';
+import { Paragraph } from '../../ui/atoms';
+import RequiredInfoTable from './RequiredInfoTable';
+
+const InitialSubmission = (): JSX.Element => {
+    const { t } = useTranslation('author-guide');
+
+    return (
+        <div className="author-guide-content">
```
Comment (will-byrne) [inline src/static-pages/components/InitialSubmission.tsx]: same as above BEM comment

File: src/static-pages/components/EditorialProcess.tsx (hunk 5 above)
Comment (norisalsaadie) [reply src/static-pages/components/EditorialProcess.tsx:65]: Yes. It's anchored to

File: src/static-pages/components/PostDecision.tsx [hunk 7]
```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Interwave from 'interweave';
+import { useTranslation } from 'react-i18next';
+import { Paragraph } from '../../ui/atoms';
+
+const PostDecision = (): JSX.Element => {
+    const { t } = useTranslation('author-guide');
+
+    return (
+        <div className="author-guide-content">
```
Comment (will-byrne) [inline src/static-pages/components/PostDecision.tsx]: Same as above BEM comment

File: src/static-pages/styles/author-guide.scss [hunk 8]
```diff
@@ -0,0 +1,40 @@
+.author-guide-page {
+    grid-area: static-nav;
+    display: flex;
+    flex-direction: column;
```
Comment (will-byrne) [inline src/static-pages/styles/author-guide.scss:4]: could this be a single column grid instead of flex?

Comment (norisalsaadie) [reply src/static-pages/styles/author-guide.scss:4]: Can be addressed as another ticket since there is one for the left hand menu.

Comment (will-byrne) [reply src/static-pages/styles/author-guide.scss:4]: ok

File: src/static-pages/components/ArticleTypes.tsx [hunk 9]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Interweave from 'interweave';
+import { useTranslation } from 'react-i18next';
+import { Paragraph } from '../../ui/atoms';
+
+const ArticleTypes = (): JSX.Element => {
+    const { t } = useTranslation('author-guide');
+
+    return (
+        <div className="author-guide__contentt">
```
Comment (will-byrne) [inline src/static-pages/components/ArticleTypes.tsx:10]: extra `t` on the end of content

**Pull request merged**